src/redisearch_rs/inverted_index/src/numeric.rs

```diff
@@ -159,17 +159,35 @@ impl ToBytes<{ size_of::<usize>() }> for Delta {
     }
 }
 
-pub struct Numeric;
+pub struct Numeric {
+    /// Should f64 values be compressed to f32 if they will only lose less than 0.01 precision
```

Comment:
```suggestion
    /// If enabled, `f64` values will be truncated to `f32`s whenever the difference is below a given threshold
```

No need to duplicate the threshold (0.01) in the comment _and_ in code.